src/app/src/sensor.actions.js

```diff
@@ -0,0 +1,29 @@
+
+import {
+    parseRiverGaugeApiData,
+    parseRiverGaugeCsvData,
+    makeRiverGaugeRequest,
+} from './sensorUtils';
+
+import {
+    startPollingSensor,
+    failPollingSensor,
+    setSensorData,
+} from './app.actions';
+
+export function pollSensor(sensor) {
+    return async (dispatch) => {
+        try {
+            dispatch(startPollingSensor());
+            const { properties: { Id, ApiAccess } } = sensor;
+            const response = await makeRiverGaugeRequest(Id, ApiAccess);
+            const sensorData = ApiAccess
+                ? parseRiverGaugeApiData(Id, response)
+                : parseRiverGaugeCsvData(Id, response);
+            return dispatch(setSensorData(sensorData));
+        } catch (e) {
+            window.console.log(e);
```

Comment:
Consider passing `e` to the `failPollingSensor` action to let Redux know there was an error. Example: https://github.com/azavea/american-water-tank-management/blob/2f8c79135f8e6137bbcefbb41b174d3338d2451c/src/app/js/actions.inspections.js#L22

I think this will fill in the `error` key that is on every payload:

![image](https://user-images.githubusercontent.com/1042475/53106474-25fba900-3501-11e9-875b-190dfd4af51c.png)